search-commons/src/main/java/no/unit/nva/search/resource/ResourceParameter.java

```diff
@@ -266,7 +240,7 @@ public enum ResourceParameter implements ParameterKey<ResourceParameter> {
     TAGS_NOT(TEXT, NOT_ALL_OF, ENTITY_TAGS),
     TAGS_SHOULD(TEXT, ANY_OF, ENTITY_TAGS),
     TAGS_EXISTS(EXISTS, ANY_OF, ENTITY_TAGS),
-    TITLE(TEXT, ENTITY_DESCRIPTION_MAIN_TITLE, PI),
+    TITLE(TEXT, ALL_OF, ENTITY_DESCRIPTION_MAIN_TITLE, PI),
```

Comment:
Is `ALL_OF` intended here?

Reply:
I removed the constructor, as it made no sense to have it, besides, I think it is better to declare `AND & OR` queries explicit.